CI/appveyor.after_success.ps1

```diff
@@ -134,52 +134,54 @@ if ("$Env:APPVEYOR_REPO_TAG" -eq "false" -and -Not $Script:PublicTestBuild) {
     $DEPLOY_URL = $DEPLOY_URL.Trim()
     echo "Deployed Mudlet to '$DEPLOY_URL'"
   } else {
-    # disabled temporarily as ssh key is unavailable
+    $sshKeyPath = "$Env:APPVEYOR_BUILD_FOLDER\CI\mudlet-deploy-key-windows"
+    ssh-keygen -t rsa -b 2048 -f $sshKeyPath -N ${Env:DEPLOY_KEY_PASS}
```

Comment:
How would authentication against the mudlet.org server work exactly when it generates a new key on the spot?

Reply:
Yeah - there already is a key at that location for which the password is available to the Core Devs...